boot/boot.js

```diff
@@ -1808,7 +1808,7 @@ $tw.loadWikiTiddlers = function(wikiPath,options) {
 	if(config["retain-original-tiddler-path"]) {
 		var output = {};
 		for(var title in $tw.boot.files) {
-			output[title] = path.relative(resolvedWikiPath,$tw.boot.files[title].filepath);
+			output[title] = path.relative(resolvedWikiPath,$tw.boot.files[title].filepath).replace(/\\/g,'/');
```

Comment:
node path.js module should take care of os specific path delimiters. ... So if this hack is needed, there is something fundamentally wrong. 

see: https://nodejs.org/api/path.html#path_path_relative_from_to

may be `path.posix.relative()` is needed here, if delimiters must always be unix style. 
see: https://nodejs.org/api/path.html#path_path_posix

Reply:
@pmario: I don't think you tried your suggestion.

On windows, I will always get paths with `\\`, but tiddler path directories (on Github) use `/`. Therefore, I need to replace the double backslashes that node returns on windows.

If you find a tested alternative that somehow has node do the translation from win to posix itself, I will gladly update the code.

Reply:
Also, I don't think there's any point in doing some `if .. else`, because on posix I will never have `\\` in a path.